lib/http/2/compressor.rb

```diff
@@ -359,15 +349,15 @@ def set_table_size(size)
       # @return [String] binary string
       def integer(i, n)
         limit = 2**n - 1
-        return [i].pack('C') if (i < limit)
+        return [i].pack('C') if i < limit
 
         bytes = []
-        bytes.push limit if !n.zero?
+        bytes.push limit unless n.zero?
```

Comment:
RuboCop's style guide suggests to avoid both:
- `if !predicate` (because it's not idiomatic ruby)
- `unless predicate; .. else` (because unless-else is hard to parse)

That's the middle ground https://github.com/bbatsov/ruby-style-guide has chosen. Does that make sense?